docs/todos.md

```diff
@@ -26,10 +26,16 @@ If you want to see todos as part of `ember-template-lint`'s output, you can incl
 ember-template-lint . --include-todo
 ```
 
-If an error is fixed manually, `ember-template-lint` will let you know that there's an outstanding todo file. You can remove this file by running `--fix`
+If an error is fixed manually, `ember-template-lint` will let you know that there's a stale todo file. You'll see this error:
 
 ```bash
-ember-template-lint . --fix
+error  Todo violation passes no-vague-rules rule. Please run `ember-template-lint /path/to/file.hbs --clean-todo` to remove this todo from the todo list.
+```
+
+You can fix this error/remove the stale todo file by running `--clean-todo`
+
+```bash
+ember-template-lint . --clean-todo
```

Comment:
Based on the example above, I think we should we replace `.` with `/path/to/file.hbs` so that it's inline with the example. You can additionally add a blurb about how to clean all todos for all files.